addons/payment_adyen/tests/test_adyen.py

```diff
@@ -41,3 +41,21 @@ def test_token_activation(self):
         token = self.create_token(active=False)
         with self.assertRaises(UserError):
             token._handle_reactivation_request()
+
+class TestAdyenNeutralize(AdyenCommon):
```

Comment:
Why a specific class ?
Since you use the same common, without any custom setup, you could reuse the same class, to avoid repeating the same setup.

(Note: same remark for all payment_* test modifs)

Reply:
It's because I think that the test class name should refelect what we test, but you are right and I will rename the classes to a more generic name.